src/pages/networks/forms/ConfigurationRow.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ReactElement, ReactNode } from "react";
+import { Button, Icon, Label } from "@canonical/react-components";
+import { CpuLimit, MemoryLimit } from "types/limits";
+import { MainTableRow } from "@canonical/react-components/dist/components/MainTable/MainTable";
+import { EditInstanceFormValues } from "pages/instances/EditInstanceForm";
+import classnames from "classnames";
+import { FormikProps } from "formik/dist/types";
+import { NetworkFormValues } from "pages/networks/forms/NetworkForm";
+import { getLxdDefault } from "util/networks";
+
+interface Props {
+  formik: FormikProps<NetworkFormValues>;
+  name: string;
+  label: string | ReactNode;
+  children: ReactElement;
+  defaultValue?: string | CpuLimit | MemoryLimit;
+  disabled?: boolean;
+}
+
+export const getConfigurationRow = ({
+  formik,
+  name,
+  label,
+  children,
+  defaultValue,
+  disabled = false,
+}: Props): MainTableRow => {
+  const values = formik.values as unknown as Record<string, string | undefined>;
+  const isOverridden = values[name] !== undefined;
```

Comment:
Override should not take the LXD default into consideration. Because it is setting the value independent of the default. I think this is quite common, that users can set a config value even though it matches the default.

For the screen capture of networks: That is expected, an auto ip address is assigning an actual IP on creation. The defaults are written into the network config by the backend, I am not sure why that is done for `ipv4/6.nat`.